### PR TITLE
Refactoring v1.1

### PR DESCRIPTION
**Pull request by jagadish-ravichandran**
1. Form validation improved
2. Removed spent column in database due to more database operations
3. Removed print statements

### CODE REVIEW

File: controller/authentication.py [hunk 1]
```diff
@@ -37,33 +37,31 @@ class RegisterForm(FlaskForm):
...
     UserService.register(email, password)
     return make_response(redirect(location=url_for('login')))
 
 
 def start_login():
     form = LoginForm()
-    template = render_template('login.html', form=form)
-
+    login_template = 'login.html'
     if not form.validate_on_submit():
```
Comment (kitrakrev) [inline controller/authentication.py:55]: is form.validate present?

File: controller/events.py [hunk 2]
```diff
@@ -33,29 +35,28 @@ def create_event():
...
+        return template
 
+    EventService.create(user_email, name, budget)
+    flash("Event added successfully", "success")
+    return redirect(url_for('event_list'))
 
 
 def show_event_list():
     user_email = current_user.email
     events = EventService.get_list(user_email)
```
Comment (kitrakrev) [inline controller/events.py:52]: what will this function return?

Comment (jagadish-ravichandran) [reply controller/events.py:52]: It redirects to the URL which is binded by the view 'event_list'.

File: controller/authentication.py (hunk 1 above)
Comment (jagadish-ravichandran) [reply controller/authentication.py:55]: if it presents, the form validation is successful and redirect to dashboard page

**Pull request merged**